glide-core/src/client/value_conversion.rs

```diff
@@ -141,9 +141,19 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
-        ExpectedReturnType::BulkString => Ok(Value::BulkString(
-            from_owned_redis_value::<String>(value)?.into(),
-        )),
+        ExpectedReturnType::BulkString => match value {
```

Comment:
Alternative way is to convert as https://github.com/valkey-io/valkey-glide/blob/98d3c5ed19039a5f5fbb8302a39fd7a017d82f5f/glide-core/src/client/value_conversion.rs#L70-L73